### DISCUSSION TITLE
no aparecen plots

### DISCUSSION THREAD

**Discussion by fermin55** [Chia-Network/chia-blockchain#8387] [Support]
buenas noches, reinicie la computadora y al instalar chia, no me aparecen mis plots, estan los directorios, pero parecen los plots en el sistema, si alguien sabe como hacer que se restablescan los plots.
gracias

**Comment by BrandtH22** [comment]
@fermin55, the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

Note - generally re-adding the plots either via the gui or cli will resolve this issue. Join us in the discord or reopen this ticket if the issue persists.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
